messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/shallow/ShallowResourceAdd.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2019 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.extension.messaging.activemq.shallow;
+
+import org.jboss.as.controller.AbstractAddStepHandler;
+import org.jboss.as.controller.AttributeDefinition;
+import org.jboss.as.controller.OperationContext;
+import org.jboss.as.controller.OperationFailedException;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.registry.Resource;
+import org.jboss.dmr.ModelNode;
+
+/**
+ *
+ * @author Emmanuel Hugonnet (c) 2019 Red Hat, Inc.
+ */
+public abstract class ShallowResourceAdd extends AbstractAddStepHandler {
+
+    protected final boolean isLegacyCall;
```

Comment:
is this field necessary? this base class does not do anything with it and it seems that it is not related to the "shallow" aspect of the resource.

Reply:
it is used in the implementations, making it private and adding a protected getter would suit you better ?

Reply:
ok moving it to the  2 implementations